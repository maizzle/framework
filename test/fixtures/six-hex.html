<div style="background: #000;">test</div>
