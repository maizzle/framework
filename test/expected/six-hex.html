<div style="background: #000000;">test</div>
